sound/soc/intel/boards/sof_sdw.c

```diff
@@ -935,6 +958,13 @@ static int mc_probe(struct platform_device *pdev)
 
 	dmi_check_system(sof_sdw_quirk_table);
 
+	if (quirk_override != -1) {
+		dev_info(&pdev->dev, "Overriding quirk 0x%x => 0x%x\n",
+			 (unsigned int)sof_sdw_quirk, quirk_override);
```

Comment:
sorry, no, I think the proper fix would be using the "%lx" format.

Reply:
yes fixed.